core/store/store.go

```diff
@@ -4,7 +4,7 @@ import dbm "github.com/cosmos/cosmos-db"
 
 // KVStore describes the basic interface for interacting with key-value stores.
 type KVStore interface {
-	// Get returns nil iff key doesn't exist. Errors on nil key.
+	// Get returns nil if key doesn't exist. Errors on nil key.
```

Comment:
Thanks for your PR.
I think `iff` is fine as it means if and only if.

Reply:
Ahh, sorry. Thank you, learned a new thing :)